winterfell/models/form.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+var _ = require('lodash');
+var FormProps = require('../utils/formProperties');
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var FormSchema = new Schema({
+  _id: String,
```

Comment:
mongoose does allow renaming of keys, but as you said, form should be a static json